### DISCUSSION TITLE
Create dvc.yaml files with less repetition with dvcrecord

### DISCUSSION THREAD

**Discussion by talosiot-will** [treeverse/dvc#6011] [Show and tell]
@dberenbaum Suggested I post this here for folks interested in this discussion.  I love DVC but didn't love having to repeat parameter and file names across source code and cli commands like `dvc run`.  [DVC Record](https://github.com/talosiot-will/dvcrecord) is a quick tool to automatically create a `dvc.yaml` file at the end of a stage run based on the parameters and files you actually used in the stage.

**Comment by karajan1001** [comment]
Wow, that's really cool! :fire:

**Reply by talosiot-will** [reply]
Thanks.  If you decide to give it a go I'd love to hear your experience.  Its something I cooked up for me and my team to use and we are still debating the best design patterns to use.
